Library/Homebrew/dev-cmd/typecheck.rb

```diff
@@ -80,6 +85,15 @@ def run
             srb_exec << "--autocorrect"
           end
 
+          if args.lsp?
+            srb_exec << "--lsp"
+            if (watchman = which("watchman", ORIGINAL_PATHS))
+              srb_exec << "--watchman-path" << watchman
+            else
+              srb_exec << "--disable-watchman" unless which("watchman", ORIGINAL_PATHS)
```

Comment:
```suggestion
              srb_exec << "--disable-watchman"
```

already in an `else` block

Reply:
Oops, good catch!